tasks/cephfs/test_client_recovery.py

```diff
@@ -307,19 +310,43 @@ def test_network_death(self):
         self.assertLess(recovery_time, self.ms_max_backoff * 2)
         self.assert_session_state(client_id, "open")
 
+
     def test_filelock(self):
         """
         Check that file lock doesn't get lost after an MDS restart
         """
-        lock_holder = self.mount_a.lock_background()
+        a_version_str = get_package_version(self.mount_a.client_remote, "fuse")
+        b_version_str = get_package_version(self.mount_b.client_remote, "fuse")
+        flock_version_str = "2.9"
+
+        version_regex = re.compile(r"[0-9\.]+")
+        a_result = version_regex.match(a_version_str)
+        self.assertTrue(a_result)
+        b_result = version_regex.match(b_version_str)
+        self.assertTrue(b_result)
+        a_version = version.StrictVersion(a_result.group())
+        self.assertTrue(a_version)
```

Comment:
Minor nitpick: I think these assertTrues on the version aren't meaningful; StrictVersion constructor is guaranteed to give you a valid object, else it would raise an exception (and afaict there's no such thing as StrictVersion that evaluates to false)